.gitignore

```diff
@@ -56,4 +56,6 @@ test/*.js
 .eslintcache
 *.swp
 *~
-
+/packages/vc-api-issuer/plugin.schema.json
```

Comment:
what is the reason these got added?

Reply:
If I remember correctly I think there was a discussion about it that these were not required.
But for now I am ommitting these as this is not integral part of this ticket.